cmd/basecmd/environmentcommand.go

```diff
@@ -282,36 +299,78 @@ func (c *EnvCommandBase) ConnectionName() string {
 	return c.envName
 }
 
+// WrapeSystemOption sets various parameters of the
+// EnvironCommand wrapper.
+type WrapEnvOption func(*environCommandWrapper)
+
+// EnvSkipFlags instructs the wrapper to skip --e and
+// --environment flag definition.
+func EnvSkipFlags() WrapEnvOption {
```

Comment:
or:

    var EnvSkipFlags WrapEnvOption = func(w *environCommandWrapper) {
        w.useDefaultEnvironment = false
    }

to save needing to call the function?